app/assets/stylesheets/application.scss

```diff
@@ -2,29 +2,27 @@
 @import 'govuk_publishing_components/govuk_frontend_support';
 @import 'govuk_publishing_components/component_support';
 
-// Components
+// Components@import 'govuk_publishing_components/govuk_frontend_support';
```

Comment:
Should this be on its own line?

Reply:
Ah yes, it's actually not needed at all... Good spot! Removing it now.

Reply:
Fixed.